File: 0506-relative-ranks/0506-relative-ranks.py
class Solution(object):
    def findRelativeRanks(self, score):
        """
        :type score: List[int]
        :rtype: List[str]
        """
        result = [""] * len(score)
        for n in range(1,len(score) + 1):
            index = score.index(max(score))
            if n == 1:
                result[index] = "Gold Medal"
            elif n == 2:
                result[index] = "Silver Medal"
            elif n == 3:
                result[index] = "Bronze Medal"
            else:
                result[index] = str(n)
            score[index] = -1

        return result

File: 0506-relative-ranks/0506-relative-ranks.java
class Solution {
    public String[] findRelativeRanks(int[] score) {
        String[] answer = new String[score.length];
        for(int i = 0; i < answer.length; i ++){
            int maximum = 0;
            for(int y = 0; y < score.length; y++){
                if(score[maximum] < score[y]){
                    maximum = y;
                }
            }
            if(i == 0){
                answer[maximum] = "Gold Medal";
            }
            else if(i == 1){
                answer[maximum] = "Silver Medal";
            }
            else if(i == 2){
                answer[maximum] = "Bronze Medal";
            }
            else{
                answer[maximum] = Integer.toString(i+1);
            }
            
            score[maximum] = -1;
        }
        return answer;
    }
}